_posts/2018-07-25-strimzi-0-5-0-release.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Strimzi 0.5.0 released!"
+date: 2018-07-25
+author: paolo_patierno
+---
+
+We are really happy to announce the new Strimzi 0.5.0 release with a lot of new awesome features!
+
+<!--more-->
+
+# Kafka as a native Kubernetes/OpenShift resource
+
+The first really cool feature is about the Cluster Operator moving from ConfigMaps to Custom Resources.
+It means that the Kafka and the Kafka Connect clusters aren't described by ConfigMaps anymore but through Custom Resources.
+Here's an example of the `Kafka` resource for deploying a Kafka cluster:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      type: ephemeral
+...
+...
+zookeeper:
+    replicas: 3
+...
+```
+
+Kafka is now a first citizen in the Kubernetes/OpenShift world and the cluster deployment can be managed through the `kubectl` and `oc` tools natively.
+For example, you can get Kafka cluster information through the following command:
+
+```
+oc describe kafka my-cluster
+```
+
+So `kafka` is now at same level as any other Kubernetes/OpenShift native resourse, for example `configmap`, `deployment`, `secret` and so on.
+The same is true for Kafka Connect cluster which is defined by a `KafkaConnect` Custom Resource.
+
+# Encryption support
+
+The communication across the deployed Kafka cluster is now encrypted using the TLS protocol and providing mutual authentication as well.
+More precisely, the following traffic that is now encrypted:
+
+* Zookeeper cluster communication
+* Kafka cluster commbunication
+* Communication between Kafka and Zookeeper
+* Communication between Topic Operator and Kafka / Zookeeper
+
+Setting up the TLS stuff is done by the Cluster Operator which generates the certificates needed for encryption.
+At same time a clients CA certificate is provided within a Secret so that the user can use it for signing clients certificates and allowing client authentication for those clients which wants to connect via TLS.
+
+# Logging configuration improvements
+
+In order to improve the logging experience, it's now possible to specify the logging level (i.e. `DEBUG`, `INFO`, ...) in the Custom Resource describing the cluster both for Kafka and Zookeeper.
+There are two different ways for doing this.
+
+The first one is about specifying the new log level "inline" in the cluster definition.
+
+```yaml
```

Comment:
IMHO ... I think it would be much better to have a real example of Kafka config ... I think the example as I did them in my post might be more readable for the user. E.g.

```
apiVersion: kafka.strimzi.io/v1alpha1
kind: Kafka
metadata:
  name: my-cluster
spec:
  kafka:
    ...
    logging:
      type: inline
      loggers:
        kafka.root.logger: "INFO,CONSOLE"
    ...
  zookeeper:
    ...
    logging:
      type: external
      name: customConfigMap
    ...
```